diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index d59ae41def51a55b654fd9ed36247d7833169aa5..faf1e4015479655ef999c4aee671fa9177ded6e3 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4182,6 +4182,7 @@ const FeatureEntry::FeatureVariation kClayBlockingVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
