diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index ce7dd2a5f7615145c44b3b9097d652c179c592ec..4be0d5520765a9a8e291a6e7fdc6aa806c1b687b 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2526,6 +2526,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (!is_chromeos_ash) {
     sources += [
