diff --git a/chrome/browser/resources/signin/profile_picker/profile_picker_main_view.html.ts b/chrome/browser/resources/signin/profile_picker/profile_picker_main_view.html.ts
index 9449d6d6b925e9723b206a46d1aace0db88e4ea8..0421c10cccd3b324d4c0e0fa7737bbb24265d387 100644
--- a/chrome/browser/resources/signin/profile_picker/profile_picker_main_view.html.ts
+++ b/chrome/browser/resources/signin/profile_picker/profile_picker_main_view.html.ts
@@ -25,7 +25,7 @@ export function getHtml(this: ProfilePickerMainViewElement) {
             data-index="${index}">
         </profile-card>
       `)}
-      <cr-button id="addProfile" class="profile-item"
+      <cr-button id="addProfile" class="profile-item plain"
           @click="${this.onAddProfileClick_}"
           aria-labelledby="addProfileButtonLabel">
         <div id="addProfileButtonLabel"
