diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 93e6339bd461d8072ba3e2cac2fa6f63e8790525..7b7f3051d789d3c72d79ffaa93965877d77ef2c6 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3540,6 +3540,7 @@ source_set("browser") {
       "profiling_utils.cc",
     ]
   }
+  import("//brave/content/browser/sources.gni") deps += brave_content_browser_deps
 }
 
 if (is_android) {
